### DISCUSSION TITLE
AC Unit

### DISCUSSION THREAD

**Discussion by Gianpaolo-Saracino** [andrei-tatar/node-red-contrib-smartnora#253] [Q&A]
Hi,
I configured an AC Unit, it seems to work fine in node-red, but there is no a rapresentation of the entity in Google Home. Or better, there is only a button in the home page that i can click...but google says that node is not configured! 
Help me please

**Comment by andrei-tatar** [comment]
@Gianpaolo-Saracino Google Home lacks UI buttons for a lot of traits.
Here's the entry in the FAQ:
https://github.com/andrei-tatar/node-red-contrib-smartnora/blob/master/doc/faq/README.md#some-devices-show-no-status-in-google-home-app-and-can-only-be-controlled-using-voice

And the list of touch control support for all the traits:
https://developers.home.google.com/cloud-to-cloud/features/touch-controls
